src/components/SideMenu/SideMenu.vue

```diff
@@ -68,34 +133,24 @@ export default {
       selectedItem: "",
       upperNav: [
         {
-          title: "Home",
+          title: "Trade Zone",
           icon: "mdi-home",
```

Comment:
just a personal preference, would be nice if we have an icon that related to trading or stock rather than a house

Reply:
example 
![image](https://user-images.githubusercontent.com/30036024/145328598-aa1b4798-ae0e-43c2-b8be-5ee1bedf8504.png)

Reply:
![image](https://user-images.githubusercontent.com/30036024/145328644-ece76f0a-c21c-4bd6-8a1c-5d8035d9afb6.png)